usr/share/rear/rescue/GALAXY11/default/600_add_galaxy_group.sh

```diff
@@ -0,0 +1,9 @@
+#
+# CommVault uses a special group which we need to copy to the rescue system
+#
+
+# append group info to /etc/group if it is not root (0)
+galaxy_group=$(stat -L -c "%g" /opt/commvault/Base64/Galaxy)
+if test $galaxy_group -gt 0 ; then
+    getent group $galaxy_group >>$ROOTFS_DIR/etc/group
```

Comment:
Only an offhanded hint (I didn't check any details here):
Perhaps our already implemented
```
CLONE_USERS=()
CLONE_GROUPS=()
CLONE_ALL_USERS_GROUPS="true"
```
provides already the needed functionality
or it could be relatively easily enhanced
to provide the needed functionality?

Reply:
Yes, however this code is about ensuring the group exists in the recovered system, not in the rescue system (what `CLONE_GROUPS` provides).

I guess this came in to avoid strange edge cases where the recovered system could not be managed via the CommVault backup server as it uses a dedicated user for remote SSH access. 

As this is legacy code we just keep copying it till somebody tells us that we don't need it anymore.

@paspflue any thoughts?

Reply:
But it appends to $ROOTFS_DIR/etc/group in the rescue system
and the recovered system should get its /etc/group restored
from the backup?
Again: I didn't check any details how that code works here.

Reply:
You are right and I was confused. To be honest, since `CLONE_ALL_USERS_GROUPS="true"` this is probably obsolete and in any case should be part of `prep` and not `rescue`. I'll fix it and thanks for asking again.